Tests/Docker/StorageApiWriterTest.php

```diff
@@ -551,11 +581,21 @@ public function testWriteTableIncrementalWithDelete()
 
         // And again, check first incremental table
         $writer->uploadTables($root . "/upload", ["mapping" => $configs]);
-        $this->client->exportTable("out.c-docker-test.table1", $root . "/download.csv");
-        $this->assertEquals(
-            "\"Id\",\"Name\"\n\"test\",\"test\"\n\"test\",\"test\"\n\"aabb\",\"ccdd\"\n",
-            file_get_contents($root . "/download.csv")
-        );
+        $this->client->exportTable("out.c-docker-test.table1", $root . DIRECTORY_SEPARATOR . "download.csv");
+        $table = $this->client->parseCsv(file_get_contents($root . DIRECTORY_SEPARATOR . "download.csv"));
+        usort($table, function ($a, $b) {
```

Comment:
imho by bylo dobrý testovat redshift a mysql odděleně, nespoléhat se na default bucket, ale explicitně založit out bucket na mysql a na redshiftu a z obou to vyzkoušet